### ISSUE TITLE
math formula not display well when converting to html

### ISSUE THREAD

**Issue by digdig** [ipython/ipython#8578]
Some of the formula not displayed when converting to html.
##See this markdown cell, many part is interpreted as strings. The `&` and `'` is not well interpreted.

![as-string](https://cloud.githubusercontent.com/assets/5459093/8362758/529ecdd2-1bad-11e5-9d4e-43dbce7b2df6.png)

##The formula display well in ipython notebook

![formula-ok](https://cloud.githubusercontent.com/assets/5459093/8362780/7f7bf370-1bad-11e5-99c1-2c97fdf4ca94.png)

## But not displaying when converting to html

![formula-bad](https://cloud.githubusercontent.com/assets/5459093/8362788/8f0a4d6e-1bad-11e5-896c-9a5cc7e4e955.png)

**Comment by Carreau** [comment]
Can you give us a string version of that not to have to retype ?

Thanks.

**Comment by digdig** [comment]
I don't know if I can send this by email attachment.

**Comment by Carreau** [comment]
Just copy past the thing in comment here, or create a gist. It's fine. And no Email attachment do not work on github.

You can also just mail it directly to one of the core dev if you don't achieve to upload our mails are not hard to find.

**Comment by digdig** [comment]
\begin{align}
E_D^{(1)} &=2\sum_{kk'}\left<kk'|V|kk'\right>
\\
&=\frac{2}{V}\sum_{kk'}\int d^3x V(x)
\\
&=\frac{N^2}{2V} \tilde{V}(0)
\end{align}

Where $\tilde{V}(0)$ is the Fourier transform $\tilde{V}(q=0)$, it is a well defined negative value. So we get: $E_D^{(1)}/N=\tilde{V}(0) n/2$.

For the exchange part of potential energy:


\begin{align}
E_X^{(1)}&=-\sum_{kk'}\left<kk'|V|k'k\right>
\\
&=\frac{1}{V}\sum_{kk'}\int d^3 x e^{i(k'-k)\cdot x}(-V(x))
\\
& <\frac{1}{V}\sum_{kk'}\int d^3 x (-V(x))
\\
& = -\frac{1}{4}Nn\tilde{V}(0)
\end{align}

Together, we get:


\begin{align}
E<\alpha n^{\frac{2}{3}}+\frac{\tilde{V}(0)}{4} n
\end{align}

This expression tell us: as $n\to \infty$, $E$ is negative and unbounded, thus the system will collapse.


### Please also note the last problem I committed, thank you very much.

**Comment by Carreau** [comment]
Yes, it is tricky as Markdown is not supposed to support math with mathjax, so there are some limitations. 

I'm already surprised that `\begin\end` works without $$...

**Comment by digdig** [comment]
Yes, it works as well as stackexchange site. All the problems are not problems in the ipython notebook interface. they only messed up when converting to html.

**Comment by digdig** [comment]
The formula I typed using `align` environment, meaning I can align the formula nicely. It seems that the interpretation of `&` in the environment is also a problem. If you just type a one line formula without `&` operator, thing get a little bit better.

**Comment by waywaaard** [comment]
We have the same problem with the following formula:

```latex
$\qquad \qquad t_n=t_0+n \cdot \Delta t \\ 
 \qquad \qquad \partial_1 = f(t_{n-1},P_{n-1})\cdot \Delta t \\
 \qquad \qquad \partial_2 = f(t_{n-1}+0.5 \cdot \Delta t,P_{n-1}+0.5\cdot\partial_1)\cdot \Delta t \\
 \qquad \qquad \partial_3 = f(t_{n-1}+0.5 \cdot \Delta t,P_{n-1}+0.5\cdot\partial_2)\cdot \Delta t \\
 \qquad \qquad \partial_4 = f(t_{n-1}+\Delta t,P_{n-1}+\partial_3)\cdot \Delta t \\
 \qquad \qquad P_n=P_{n-1}+(\partial_1+2\cdot\partial_2+2\cdot\partial_3+\partial_4)/6
 $
```

It is working in the notebook, but not for the export. I guess you need to apply the same fixes as for the  javascript markdown parser.

**Comment by waywaaard** [comment]
Still not working 3.2.1

**Comment by minrk** [comment]
@waywaaard you probably want to use display math instead of inline math for a big multi-line collection of equations. You would use `$$` instead of `$` for that.
